src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -62,16 +68,23 @@
 public class SkroutzGreekStemmer {
   public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
   private final CharArraySet stopwords;
+  private Map<String,char[]> step_0_exceptions;
 
   public SkroutzGreekStemmer(final CharArraySet stopwords) {
     this.stopwords = stopwords;
+    this.step_0_exceptions = new HashMap(0);
```

Comment:
This is being called in two cases:

1. When we are filtering out stopwords. Here we are initializing it and doing nothing with it.
2. In exceptional cases (I/O failures etc.) when trying to read the YAML file that contains the stemming exception for level 0. Again initializing it just to avoid any null pointers and do nothing with it.

This map is not being used later on.
@cmantas could verify this.